src/main/java/com/project/momskitchen/backend/persistence/impl/MomsKitchenDB.java

```diff
@@ -0,0 +1,53 @@
+package com.project.momskitchen.backend.persistence.impl;
+
+import java.sql.*;
+import com.project.momskitchen.backend.exceptions.MomsPersistenceException;
+import com.project.momskitchen.backend.model.User;
+
+public class MomsKitchenDB {
+    //Direccion del DB de prueba
+    static String urlDatabase = "";
+    static String usuarioDb = "";
+    static String passwordDb= "";
+    static Connection c = null;
+
+    public void realizaConexion() throws MomsPersistenceException{
+
+        try {
+            Class.forName("org.postgresql.Driver");
```

Comment:
If you use Spring Data it is likely that you don't need to implement a method to establish the connection with your database